vignettes/example1.Rmd

```diff
@@ -257,25 +260,25 @@ model <- stan_glm(weight ~ feed, data=data)
 posteriors <- insight::get_parameters(model)
 
 ggplot(posteriors, aes(x=feedsunflower)) +
-  geom_density(fill = "red")
+  geom_density(fill = "wheat")
 ```
 
-This represents the **posterior distribution of the difference between meat meals and Sunflowers**. Seems that the difference is rather **positive** (the values seems concentrated on the right side of 0)... Eating Sunflowers makes you more fat (*at least, if you're a chicken*). **By how much?** Let us compute the **median** and the **CI**:
+This represents the **posterior distribution of the difference between `meatmeal` and `sunflowers`**. Seems that the difference is rather **positive** (the values seems concentrated on the right side of 0)... Eating sunflowers makes you more fat (*at least, if you're a chicken*). But, **by how much?** Let us compute the **median** and the **CI**:
 
 ```{r message=FALSE, warning=FALSE, eval=TRUE}
 median(posteriors$feedsunflower)
 hdi(posteriors$feedsunflower)
 ```
 
-It makes you fat by around `51` (the median). However, the uncertainty is quite high: **there is 89\% chance that the difference between the two feed types is between `7.77` and `87.66`.**
+It makes you fat by around `51` grams (the median). However, the uncertainty is quite high: **there is 89\% chance that the difference between the two feed types is between `7.77` and `87.66`.**
 
 > **Is this effect different from 0?**
 
 ### ROPE Percentage
 
 Testing whether this distribution is different from 0 doesn't make sense, as 0 is a single value (*and the probability that any distribution is different from a single value is infinite*). 
 
-However, one way to assess **significance** could be to define an area around 0, which will consider as *equivalent* to zero (*i.e.*, absence of, or negligible, effect). This is called the [**Region of Practical Equivalence (ROPE)**](https://easystats.github.io/bayestestR/articles/region_of_practical_equivalence.html), and is one way of testing the significance of parameters.
+However, one way to assess **significance** could be to define an area around 0, which will consider as *practically equivalent* to zero (*i.e.*, absence of, or negligible, effect). This is called the [**Region of Practical Equivalence (ROPE)**](https://easystats.github.io/bayestestR/articles/region_of_practical_equivalence.html), and is one way of testing the significance of parameters.
```

Comment:
I am not sure here, as the practically might be redundant and confusing for non-english (in French, "practically" (pratiquement) refers both to "in a practical manner" and to "almost"). Hence, keeping here only equivalent avoids this problem (and the practically comes afterwards either way.

Reply:
We could keep practically here, as ROPE also has it in its name. I think here the redundancy serves a better understanding.

Reply:
I've kept in f047f6ff655ce87849a2a3ec23e33d88f14342aa

Reply:
alright